docs/guide/http_cache.md

```diff
@@ -600,90 +617,92 @@ To avoid overloading any application code, you will take advantage of Symfony's
 
-In case of complex content, for instance Landing Pages with many blocks, or RichText with a lot of embeds/links, you
-can encounter problems with too long cache header on responses. Because of this, necessary cache entries may not be
-tagged properly. You may also see `502 Headers too long` errors, and webserver refusing to serve the page.
+In case of complex content, for instance Pages with many blocks, or RichText with a lot of embeds/links,
+you can encounter problems with too long cache header on responses.
+Because of this, necessary cache entries may not be tagged properly.
+You may also see `502 Headers too long` errors, and webserver refusing to serve the page.
 
 Here are some options on how to solve this issue:
 
-**A. Configure to allow larger header:**
+###### A. Configure  allowing larger headers
 
-**Varnish** config:
+Varnish configuration:
 
 - [http_resp_hdr_len](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-hdr-len) (default 8k, change to i.e. 32k)
 - [http_max_hdr](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-max-hdr) (default 64, change to i.e. 128)
 - [http_resp_size](https://varnish-cache.org/docs/6.0/reference/varnishd.html#http-resp-size) (default 23k, change to i.e. 96k)
 - [workspace_backend](https://varnish-cache.org/docs/6.0/reference/varnishd.html#workspace-backend) (default 64k, change to i.e. 128k)
 
-If you need to see these long headers in the `varnishlog` adapt the [vsl_reclen](https://varnish-cache.org/docs/6.0/reference/varnishd.html#vsl-reclen) setting.
+If you need to see these long headers in `varnishlog`, adapt the [vsl_reclen](https://varnish-cache.org/docs/6.0/reference/varnishd.html#vsl-reclen) setting.
 
-**Nginx** has a default limit of 4k/8k when buffering responses:
+Nginx has a default limit of 4k/8k when buffering responses:
 
 - For [PHP-FPM](https://www.php.net/manual/en/install.fpm.php) setup using proxy module, configure [proxy_buffer_size](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size)
 - For FastCGI setup using fastcgi module, configure [fastcgi_buffer_size](https://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_buffer_size)
 
-**Fastly** has a `Surrogate-Key` header limit of 16 kB, this cannot be changed.
+Fastly has a `Surrogate-Key` header limit of 16 kB, this cannot be changed.
 
-**Apache** has a [hard](https://github.com/apache/httpd/blob/5f32ea94af5f1e7ea68d6fca58f0ac2478cc18c5/server/util_script.c#L495) [coded](https://github.com/apache/httpd/blob/7e2d26eac309b2d79e467ef586526c10e0f226f8/include/httpd.h#L299-L303) limit of 8 kB, so if you face this issue consider using Nginx instead.
+Apache has a [hard](https://github.com/apache/httpd/blob/5f32ea94af5f1e7ea68d6fca58f0ac2478cc18c5/server/util_script.c#L495) [coded](https://github.com/apache/httpd/blob/7e2d26eac309b2d79e467ef586526c10e0f226f8/include/httpd.h#L299-L303) limit of 8 kB, so if you face this issue consider using Nginx instead.
```

Comment:
```suggestion
Apache has a [hard](https://github.com/apache/httpd/blob/5f32ea94af5f1e7ea68d6fca58f0ac2478cc18c5/server/util_script.c#L495) [coded](https://github.com/apache/httpd/blob/7e2d26eac309b2d79e467ef586526c10e0f226f8/include/httpd.h#L299-L303) limit of 8 kB, so if you face this issue, consider using Nginx instead.
```
Two external links linked from each word of the "hard coded" phrase. Could it possibly be split somehow?